http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1120,7 +1124,7 @@ class LeoPubsubMessageSubscriber[F[_]](
               .getDisk(
                 msg.project,
                 zone,
-                d.name
+                GoogleDiskName(d.name.asString)
```

Comment:
Why did we make `AzureDiskName` be `DiskName` and then add `GoogleDiskName`. Can we not use the same for both? I don't understand how the same field in a database can be decoded to 2 different value classes..